sway-lib-std/src/tx.sw

```diff
@@ -166,6 +166,17 @@ pub fn get_script_data<T>() -> T {
     }
 }
 
+/// Get the script bytecode
+/// Must be cast to a u64 array, with sufficient length to contain the bytecode.
+/// Bytecode will be padded to next whole word.
+pub fn tx_script_bytecode<T>() -> T {
+    let script_ptr = tx_script_start_offset();
+    let script_bytecode = asm(r1: script_ptr) {
+        r1: T
+    };
+    script_bytecode
```

Comment:
```suggestion
   asm(r1: script_ptr) {
        r1: T
    }
```

Reply:
Resolved in [a7c7a91](https://github.com/FuelLabs/sway/pull/2142/commits/a7c7a91776162bd5d4a0bbbbf94c14c066c83e9a)